### DISCUSSION TITLE
TOC feature and limitation of code block height not working in Rmd files

### DISCUSSION THREAD

**Discussion by jantau** [chipzoller/hugo-clarity#133] [General]
Hi Chip, thanks for the Hugo theme. It is great and I like it a lot. 

Some features that are working in md files are not working in Rmd files. For example the TOC option, the limitation of code block height or the read more tag. Is there a way to make these features work in Rmd files?

Thank you!

**Comment by chipzoller** [comment]
Hi there. I don't think we tested these with RMD files. I recall a similar request at one point.

**Reply by onweru** [reply]
> I recall a similar request at one point.

That would be #75

**Reply by jantau** [reply]
Thanks for your reply and the reference to issue #75. Next time I will screen the issues first. I only looked at the discussion board this time.
